client/app/containers/test-run-info/logs-agents/default.logs-agent.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const defaultLogsAgent = function defaultLogsAgent(
+    authService,
+    moment,
+) {
+    'ngInject';
+
+    const fileExtensionPattern = /(\.[0-9a-z]+)(?:[\?#]|$)/i;
+
+    return {
+        name: 'Default logs agent',
+        esIndex: '',
+        prefix: 'test-run-data-',
+        searchCriteria: [],
+
+        initESIndex,
+        initSearchCriteria,
+        handleScreenshotLog,
+    };
+
+    function handleScreenshotLog(log) {
+        log.level = 'INFO';
+        log.urls = {};
+
+        if (log.message) {
+            log.originalMessage = log.message;
+            log.message = 'Screenshot is captured';
+
+            const relativeImageUrl = log.originalMessage
+                .replace(new RegExp(`^(${log.tenant || authService.tenant})`, 'i'), '');
+
+            if (relativeImageUrl) {
+                const imageUrl = `${authService.serviceUrl}${relativeImageUrl}`;
+                const thumbnailUrl = imageUrl.replace(fileExtensionPattern, '_thumbnail$1');
+
+                log.urls.image = { path: imageUrl, name: 'screenshot', }
```

Comment:
it's bad to add comma after the name property. Only if you use new line for each property